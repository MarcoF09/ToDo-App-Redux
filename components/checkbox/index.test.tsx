import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import React from 'react'
import { CheckBox } from './index'

describe('should render button', () => {
  it('renders correctly with checked', () => {
    const component = shallow(<CheckBox checked={true} click={jest.fn()} />)
    expect(toJson(component)).toMatchSnapshot()
  })
  it('renders correctly without checked', () => {
    const component = shallow(<CheckBox checked={false} click={jest.fn()} />)
    expect(toJson(component)).toMatchSnapshot()
  })
})
