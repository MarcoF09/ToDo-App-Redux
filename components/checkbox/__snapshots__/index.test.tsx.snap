// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render button renders correctly with checked 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-end",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[MockFunction]}
  >
    <Image
      source={
        Object {
          "testUri": "../../../resources/images/iconCheckboxActive.png",
        }
      }
    />
  </TouchableOpacity>
</View>
`;

exports[`should render button renders correctly without checked 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-end",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[MockFunction]}
  >
    <Image
      source={
        Object {
          "testUri": "../../../resources/images/iconCheckboxInactive.png",
        }
      }
    />
  </TouchableOpacity>
</View>
`;
